src/Services/InteractWithPage.php

```diff
@@ -54,10 +54,24 @@ protected function see($text, $tag = 'body')
         return $this;
     }
 
+    /**
+     * @param $text
+     * @param string $tag
+     */
     protected function notSee($text, $tag = 'body')
     {
         $this->assertNotContains($text, $this->byTag($tag)->text());
     }
+     
+    /**
+     * alias of notSee.
+     * @param $text
+     * @param string $tag
+     */
+    protected function dontSee($text, $tag = 'body')
+    {
+        $this->notSee($text, $tag);
```

Comment:
Replace this with ` $this->assertNotContains($text, $this->byTag($tag)->text());` because we are going to deprecate `notSee` method.